docs/filesystem_layer/stratum1.md

```diff
@@ -87,14 +95,82 @@ You can request a firewall exception rule to be added for your Stratum 1 server
 
 Make sure to include the IP address of your server.
 
+## Step 3: Verification of the Stratum 1
+
+When the playbook has finished your Stratum 1 should be ready. In order to test your Stratum 1, even
+without a client installed, you can use curl.
+
+```bash
+curl --head http://url-to-your-stratum1/cvmfs/pilot.eessi-hpc.org/.cvmfspublished
+```
+This should return:
+
+```bash
+HTTP/1.1 200 OK
+...
+X-Cache: MISS from url-to-your-proxy
+```
+
+The second time you run it, you should get a cache hit:
+
+```bash
+X-Cache: HIT from url-to-your-proxy
+```
+
+Example with the Norwegian Stratum 1:
+
+```bash
+curl --head http://bgo-no.stratum1.cvmfs.eessi-infra.org/cvmfs/pilot.eessi-hpc.org/.cvmfspublished
+```
+
+You can also test access to your Stratum 1 from a client
+
+To test it you will have to install the CMVFS
+[`client`](https://github.com/EESSI/filesystem-layer#clients).
```

Comment:
```suggestion
You can also test access to your Stratum 1 from a client, for which you will have to install the CVMFS
[client](https://github.com/EESSI/filesystem-layer#clients). 
```

Reply:
This should be moved to the documentation as well, but for now this link is okay. We can change that later on.